examples/compareKernelTypes.py

```diff
@@ -49,70 +51,70 @@
 defSciencePath = None
 defTemplatePath = None
 
+
 class DiffimTestCases(unittest.TestCase):
```

Comment:
Do they actually run? Consult with @parejkoj and @r-owen as to whether to move them or leave them as is.